main.go

```diff
@@ -38,7 +40,7 @@ func setLogStreams(
 }
 
 func main() {
-	//Init Log streams
+	//Setup log streams
```

Comment:
Space at start of comments?

Reply:
Is that something you want me to change?

Reply:
What's the style in Go?

Reply:
The style is to have a space, I'll add it